assets/in

```diff
@@ -100,6 +100,7 @@ args="$args -Drepository.url=$url"
 [ -n "$password" ] && args="$args -Drepository.password=$password";
 
 $resource_dir/mvnw dependency:copy $args
+echo $version > version
```

Comment:
To get this to work for me I had to use:
```
echo "$version" > "$destination/version"
```

I also found adding the following to be useful:
```
echo "$artifactItem" > "$destination/artifact"
```

Reply:
We are using my fork with these same changes in a few pipelines that use the version file with no issue.
Any idea why this isn't working for you?

Reply:
That does seem like the safer/better way to refer to the file. I'll update the pull request when I get a chance

Reply:
That's a good question, I'm really not sure; I didn't get a chance to really dig into it. We're running Concourse 3.13.0, and it never seemed to work (but it passed  all the tests).

Reply:
Could be the concourse version. I tested it on 4.xx and we're also on 4.xx at the moment. I want to say that someone at work is using it on 3.xx but I'm not sure. I'll take a look tomorrow because I'm curious.

Reply:
I made the change to use $destination for version file. I'm thinking the artifact file should be a separate issue/pr.